e2e/e2etests/e2etests.go

```diff
@@ -132,6 +132,7 @@ const (
 	TestMigrateERC20CustodyFundsName  = "migrate_erc20_custody_funds"
 	TestMigrateTSSName                = "migrate_TSS"
```

Comment:
```suggestion
	TestMigrateTSSName                = "migrate_tss"
```
Not related to this PR but just easier to update here instead of opening a new PR